Podcast/EpisodeSubjectView.swift

```diff
@@ -0,0 +1,237 @@
+//
+//  EpisodeFeedElementSubjectView.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 10/7/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+protocol EpisodeSubjectViewDelegate: class {
+    func episodeSubjectViewDidPressPlayPauseButton(episodeSubjectView: EpisodeSubjectView)
+    func episodeSubjectViewDidPressRecommendButton(episodeSubjectView: EpisodeSubjectView)
+    func episodeSubjectViewDidPressBookmarkButton(episodeSubjectView: EpisodeSubjectView)
+    func episodeSubjectViewDidPressTagButton(episodeSubjectView: EpisodeSubjectView, index: Int)
+    func episodeSubjectViewDidPressMoreActionsButton(episodeSubjectView: EpisodeSubjectView)
+}
+
+class EpisodeSubjectView: FeedElementSubjectView {
+    
+    static let episodeSubjectViewHeight: CGFloat = 253
+    static let episodeUtilityButtonBarViewHeight: CGFloat = EpisodeUtilityButtonBarView.height
+    
+    ///
+    /// Mark: View Constants
+    ///
+    var seperatorHeight: CGFloat = 9
```

Comment:
You spelled "separator" wrong in a few places haha

Reply:
She does this all the time

Reply:
LOL. A file probably hasn't been written by me if all the spelling is correct. I did a replace all to fix.